src/OpenSearch.Stack.ArtifactsApi/OpenSearch.Stack.ArtifactsApi.csproj

```diff
@@ -7,7 +7,9 @@
     
     <ItemGroup>
         <PackageReference Include="SemanticVersioning" Version="0.8.0" />
-        <PackageReference Include="System.Text.Json" Version="4.6.0" />
+        <PackageReference Include="System.Net.Http" Version="4.3.4" />
+        <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
```

Comment:
This looks added, why?

Reply:
These are essential packages, compilation fails without them.
I'm also surprised how it was working before.

Reply:
Eh 🤔